pages/mesosphere/dcos/services/elastic/2.7.0-6.8.1/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: layout.pug
+navigationTitle: Elastic 2.7.0-6.8.1
+excerpt: DC/OS Elastic lets you manage an Elasticsearch cluster
+title: Elastic 2.7.0-6.8.1
+menuWeight: 1
```

Comment:
Since this is now the latest version can you delete the entire directory for `2.5.0-6.3.2` so that the only pages that show up in service docs are `2.7.0-6.8.1` and `2.6.0-6.6.1`? Same thing will happen when we add Elastic 7.3 which will represent the latest major and n-1 versions.